src/mysql_vm_helpers.py

```diff
@@ -455,12 +455,15 @@ def _execute_commands(
             bash: whether to run the commands with bash
             user: the user with which to execute the commands
             group: the group with which to execute the commands
-            env: the environment variables to execute the commands with
+            env_extra: the environment variables to add to the current process’ environment
 
         Returns: tuple of (stdout, stderr)
 
         Raises: MySQLExecError if there was an error executing the commands
         """
+        env = os.environ.copy()
```

Comment:
i wonder if there are env vars like `$HOME` or `$USER` which we should not be passing to subprocess (as they may result in permissions issues)